src/lib.rs

```diff
@@ -4,7 +4,7 @@
 // 1. Run `rustc -W help`
 // 2. Grab all `default=warn` warnings
 // 3. Paste them here, deleting `warnings`, and any with `deprecated` in the name
-#![deny(const_err,
+#![cfg_attr(not(feature="dev"), deny(const_err,
```

Comment:
- If we're fixing it here, should we also fix it in the bin?
- Should the "fix" be to get it working again or remove it completely?  How does this impact your workflow?

Reply:
oh yeah, I missed out the bin.
It wasn't really impacting me, I've noticed this by chance and as it was supposed to be ignored in feature "dev" I made it works again :)

Reply:
RE fixing this

I guess my question is more broad, of if anyone cares about the `dev` feature and if not, if we should just remove it.

Reply:
I'm using it sometimes when moving stuff around and experimenting, this phase can left a lot of unused import or result not used but only temporarily.

Reply:
Thanks for explaining your case.

I guess I'm fine living with the task master of warnings during my experimentation but I shouldn't force that on others :)